src/ncp/ncp_base.cpp

```diff
@@ -1288,16 +1288,6 @@ otError NcpBase::HandlePropertySet_SPINEL_PROP_NEST_STREAM_MFG(uint8_t aHeader)
     VerifyOrExit(error == OT_ERROR_NONE, error = WriteLastStatusFrame(aHeader, ThreadErrorToSpinelStatus(error)));
 
     output[sizeof(output) - 1] = '\0';
-
-#if OPENTHREAD_MTD || OPENTHREAD_FTD
```

Comment:
https://github.com/openthread/wpantund/pull/432 has been merged.

Reply:
Should we keep this for the time being for backward-compatibility purposes?